go/ql/test/query-tests/InconsistentCode/UnhandledCloseWritableHandle/tests.go

```diff
@@ -40,41 +27,51 @@ func closeFileDeferredIndirectReturn(f *os.File) {
 }
 
 func deferredCalls() {
-	if f, err := openFileWrite("foo.txt"); err != nil {
+	// open file for writing
+	if f, err := os.OpenFile("foo.txt", os.O_WRONLY|os.O_TRUNC|os.O_CREATE, 0666); err != nil {
 		closeFileDeferred(f)         // NOT OK
 		closeFileDeferredIndirect(f) // NOT OK
+		closeFileDeferredIndirectReturn(f)
```

Comment:
Note whether this case is OK, not ok, spurious, etc. My guess is spurious since the close call doesn't drop the error return, but then we defer a wrapper and so drop it later.